rust/src/lib/ifaces/ovs.rs

```diff
@@ -659,6 +659,11 @@ pub struct OvsDpdkConfig {
     /// Must be power of 2 in the range of 1 to 4096.
     /// Setting to 0 means remove this setting from OVS database.
     pub n_txq_desc: Option<u32>,
+    #[serde(skip_serializing_if = "Option::is_none", rename = "lsc-interrupt")]
+    /// Configures the Link State Change (LSC) detection mode for dpdk ports.
+    /// Setting to true enables interrupt; false disables interrupt and configures
+    /// the interface in poll mode.
+    pub lsc_interrupt: Option<bool>,
```

Comment:
I think that `rename=` is not needed, `_` is replaced for `-` automatically, if I'm not wrong.

Reply:
Or at lease create an alias for `dpdk-lsc-interrupt`.